scripts/run_design.py

```diff
@@ -51,9 +52,12 @@ def main():
     # Get the flow object
     flow = flows.Flows(args.flow)
 
+    # Get the flow arguments
+    flow_args = args.flow_args
```

Comment:
I think we talked about making flow_args a dictionary so that the code here would look something like this:
```
flow_args[SYNTH_STAGE] = args.synth_args
flow_args[IMPL_STAGE] = args.impl_args
...
```

Reply:
And it would be good to make it like this so that we don't get KeyErrors if a user doesn't provide arguments for a certain type:
```
from collections import defaultdict
flow_args = defaultdict(str)
```